app/controllers/api/people_controller.rb

```diff
@@ -92,15 +92,10 @@ def validate_nomis_profile
       profile_validator.validate!
     end
 
-    def other_included_relationships
-      # Custom included relationships to avoid the people actions relationships
-      @other_included_relationships ||= IncludeParamHandler.new(params).call || OTHER_SUPPORTED_RELATIONSHIPS
-    end
-
     def validate_other_include_params
```

Comment:
Is this method still necessary?

Reply:
Yea unfortunately we have to have this custom method  (now at-least after the change) as we can support both v1/v2